### PR TITLE
Persistence and validation fixes

### PR DESCRIPTION
**Pull request by bmallred**
**/me jedi** _Just a collection of fixes for persistence and validator logic_

A lot of validation logic needed to be adjusted to account for the new structures and other changes which occurred in parallel to the initial data refactoring.

Also, used this debug the deployment issues we were having. The results were we had pushed a migration script to add new accounts before the merge of the persistence tables. This migration's version number was higher than most of the persistence table migrations _except_ the submission migration. So because the migration tool thought it had only one migration to run and thought it had applied the other migrations the deployment was crashing.

### CODE REVIEW

**Comment by floresj** [comment]
Migration works as expected:
selection_553

**Comment by floresj** [comment]
Validations :tada: 
selection_554

File: src/components/Section/Relationships/RelationshipStatus/CivilUnion.jsx [hunk 1]
```diff
@@ -411,7 +411,7 @@ export default class CivilUnion extends ValidationElement {
               <Field title={i18n.t('relationships.civilUnion.heading.addressSeparated')}
                      className="address-separated"
                      scrollIntoView={this.props.scrollIntoView}>
-                <NotApplicable name="OtherNameNotApplicable"
+                <NotApplicable name="AddressSeparatedNotApplicable"
```
Comment (bmallred) [inline src/components/Section/Relationships/RelationshipStatus/CivilUnion.jsx]: Nice find

File: src/validators/civilunion.js [hunk 2]
```diff
@@ -55,15 +55,15 @@ export default class CivilUnionValidator {
...
+    if (this.addressSeparatedNotApplicable.applicable) {
       addressValid = new LocationValidator(this.addressSeparated).isValid()
     }
 
     return validDateField(this.dateSeparated) && addressValid
   }
 
   validForeignBornDocument () {
-    if (new LocationValidator(this.birthPlace).isValid() && this.birthPlace.country !== 'United States') {
+    if (new LocationValidator(this.birthPlace).isValid() && this.birthPlace.country.value !== 'United States') {
```
Comment (bmallred) [inline src/validators/civilunion.js]: May need to `import { countryString } from 'path/to/validators/location'` and then do `countryString(this.birthPlace.country) !== 'United States')`. I've been using that method to account for the different types of values in `country`

Comment (floresj) [reply src/validators/civilunion.js]: Ahh nice. Will do! 👍

File: src/views/Login/preload.js [hunk 3]
```diff
@@ -0,0 +1,303 @@
+import { navigationWalker } from '../../config'
+import { updateApplication, reportCompletion } from '../../actions/ApplicationActions'
+import { api } from '../../services'
+import schema from '../../schema'
+
+// To run this put the following reference in Login.jsx
```
Comment (bmallred) [inline src/views/Login/preload.js]: This file is for when we need to be able to load data quickly from DevTools. Should be removed once we have a steady persistence model

**Pull request merged**